.github/workflows/build.yml

```diff
@@ -1,10 +1,9 @@
 name: ci
 on:
   pull_request:
-    paths:
-    - '**'
-    - '!doc/contributors/**'
-    - '!doc/design/**'
+    paths-ignore:
+    - 'doc/**'
```

Comment:
`./doc/install/local-values.yaml` used to be used in manifest generation, not sure if that's still true though.